app/react/Documents/components/scss/showToc.scss

```diff
@@ -0,0 +1,52 @@
+@import "../../../App/scss/config/_colors.scss";
+
+.toc {
+  padding-left: 5px !important;
+  padding-right: 5px !important;
+
+  ul.toc-view {
+    $indents: 5;
+
+    @for $i from 0 through $indents {
+      li.toc-indent-#{$i} {
+        border-radius: 2px;
+        height: 28px;
+
+        &:hover {
+          background-color: #E2E7F4;
+;
+        }
+        a:hover {
+          text-decoration: none;
+        };
+        a {
+          padding-left: 4px;
+
+          @if $i == 0 {
+            font-weight: 500;
+            font-size: 16px;
+            line-height: 18.75px;
+            color: #444;
+          } @else if $i == 1 {
+            font-weight: 400;
+            font-size: 14px;
+            line-height: 16.41px;
+            color: #333;
+          } @else if $i == 2 {
+            font-weight: 400;
+            font-size: 14px;
+            line-height: 16.41px;
+            color: #555;
```

Comment:
If we are going to have `if`s here, was it really necessary the `for`?  This is a very complex CSS to read. If we had hundreds of indents, maybe it could be worth it.  But for 5, maybe just putting:
```
li.toc-indent-1, 
li.toc-indent-2,
li.toc-indent-3,
li.toc-indent-4,
li.toc-indent-5 {
...
```

was enough?   Not sure here.

Reply:
@grafitto can you check this please ?

Reply:
True, I will just get rid of the loop all together

Reply:
Removed the loop, actually there are only 3 levels not 5